.github/workflows/openvex.yaml

```diff
@@ -0,0 +1,35 @@
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        type: string
+        required: true
+        description: "GitHub release tag"
+
+permissions: {}
+
+jobs:
+  vexctl:
+    permissions:
+      contents: write
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Set env
+        run: echo "RELEASE_VERSION=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
+      - name: Set tag name
+        shell: bash
+        run: |
+          TAG=${{ github.event.inputs.tag }}
+          echo "TAG=$TAG" >> "$GITHUB_ENV"
+      - uses: openvex/generate-vex@31b415924ea0d72ed5f2640f1dee59dea6c2770b
+        name: Run vexctl
+        with:
+          product: pkg:generic/karpenter@${{ env.RELEASE_VERSION }}
+          file: /tmp/karpenter.vex.json
+      - name: Upload openvex data
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
sanity check: Do we need to set this environment variable here or is the token just injected into the context when you are making calls in the bash script?

Reply:
I made the changes mentioned in the review and tried triggering the [workflow in my fork](https://github.com/sreeram-venkitesh/karpenter/actions/runs/7323345505). It was able to [generate and upload the json file](https://github.com/sreeram-venkitesh/karpenter/releases/tag/v0.34.9) without me having to set any environment variable.

Reply:
Thanks @sreeram-venkitesh! One comment on how we should set our environment variables.